apps/right-to-rent-check/behaviours/get-declarer.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    if (req.sessionModel.get('representative') === 'landlord') {
+      locals.declarer = req.translate('pages.declaration.landlord');
+      locals.declarerId = 'landlord';
+      locals.declarerEmail = req.sessionModel.get('landlord-email-address');
+    } else if (req.sessionModel.get('representative') === 'agent') {
+      locals.declarer = req.translate('pages.declaration.agent');
+      locals.declarerId = 'agent';
+      locals.declarerEmail = req.sessionModel.get('agent-email-address');
+    }
+    return locals;
```

Comment:
It's  small thing, but in this function you do `req.sessionModel.get('representative')` twice.
IMO, I think it would be better if you declared `const rep = req.sessionModel.get('representative')` above the `if` and used the variable.

Also, are there only two values that `req.sessionModel.get('representative')` could be?
If so, you can remove the `else if` and do ...

```js
if (req.sessionModel.get('representative') === 'landlord') {
  locals.declarer = req.translate('pages.declaration.landlord');
  locals.declarerId = 'landlord';
  locals.declarerEmail = req.sessionModel.get('landlord-email-address');
  return locals;
}
locals.declarer = req.translate('pages.declaration.agent');
locals.declarerId = 'agent';
locals.declarerEmail = req.sessionModel.get('agent-email-address');
return locals;
```

Reply:
Otherwise, it's looking good.

Reply:
good point about using a `const`

On the second point, by doing the code you've suggested, to me it looks like `locals.declarer` would be overwritten to whatever the agent is after the `if`.  However, I could just use an `else` rather than `else if` which would make it neater

Reply:
it wouldn't be overwritten because `return locals;` immediately stops execution of that function

Reply:
ahh I didn't see that, nice one Joe 👍